feature/system/management_vrf/README.md

```diff
@@ -0,0 +1,85 @@
+    *   Configure a `::/0` static route configured in the management VRF with route preference 220 going via OOB interface. This will be the backup channel for management traffic.
+*   Configure a v6 eBGP between DUT port-1 and ATE port-1.
+*   Configure a v6 eBGP between DUT port-2 and ATE port-2.
+*   Configure ATE to advertise a prefix (say, [2001:DB8:0::/64](https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md#ipv6)) to DUT.
+
+## Tests
+
+### Test-1, routing protocol validation
+
+Via gNMI, validate that:
+
+*   The configured 2 BGP sessions are up and receive the advertised `2001:DB8:0::/64/126` route on DUT.
+*   The `::/0` static route is of the expected route preference value.
+
+### Test-2, service available via linecard interfaces
+
+1.  Via gNMI, shutdown all OOB ports so that port-11 and port-12 are the only ingress ports to manage the device. This step itself validates the gNMI service availability.
+2.  Issue a gRIBI Flush() and expect a successful response.
+3.  Issue a gNOI Ping() test to ping itself and expect a successful response.
+4.  Issue a P4RT `MasterArbitrationUpdate` with random `device_id`, and expect a `StreamMessageResponse` response (no need to check the correctness).
+5.  sFLOW will be tested in its own feature profile test.
+6.  Use Ondatra raw API to run CLI command to refresh the time via NTP.  This also validates that the SSH is working.
+7.  Use Ondatra raw API to run CLI command to lookup google.com.
+8.  Syslog should be tested in its own feature profile test.
+9.  Unshut OOB ports
+
+### Test-3, service available after CONTROLLER CARD switchover
+
+1.  Via gNMI, shutdown all OOB ports so that port-11 and port-12 are the only ingress ports to manage the device.
+2.  Use gNOI command to switch over the CONTROLLER CARD. It is expected that this step probably will disconnect/disrupt existing services.
+3.  Make sure all services are still accessible within 60s, by repeating service validation steps in scenario#2.
+4.  Unshut OOB ports.
+
+### Test-4, service available via OOB ports
+
+1.  Shutdown port-11 and port-12. This ensures that the device can only be managed at the same loopback interface but via the OOB ports. This step itself validates the gNMI service availability.
+2.  Issue a gRIBI Flush() and expect a successful response.
+3.  Issue a gNOI Ping() test to ping itself and expect a successful response.
+4.  Issue a P4RT `MasterArbitrationUpdate` with random `device_id`, and expect a `StreamMessageResponse` response (no need to check the correctness).
+5.  Use Ondatra raw API to run CLI command on the DUT (e.g. show version). This validates that the SSH is working.
+6.  Unshut port-11 and port-12.
```

Comment:
```suggestion
6.  Unshut port-11 and port-12.

### test-5, service available via OOB of BACUP ONTROLER_CARD

1.  Shutdown port-11 and port-12 and OOB port of ACTIVE CONTROLER_CARD. This ensures that the device can only be managed at the same loopback interface but via the OOB ports of inactine ONTROLER_CARD. This step itself validates the gNMI service availability.
2.  Issue a gRIBI Flush() and expect a successful response.
3.  Issue a gNOI Ping() test to ping itself and expect a successful response.
4.  Issue a P4RT `MasterArbitrationUpdate` with random `device_id`, and expect a `StreamMessageResponse` response (no need to check the correctness).
5.  Use Ondatra raw API to run CLI command on the DUT (e.g. show version). This validates that the SSH is working.
6.  Unshut port-11 and port-12 and OOB port of ACTIVE ONTROLER_CARD

### Test-6, loss of routing on linecard interfaces

1. delete `::/0` routing static routes configured in the management VRF, with default route preference and next hops being the adjacent IP of port-11 and port-12.\
(This emulates eBGP session failure in prod setup).
This step itself validates the gNMI service availability.
2.  Issue a gRIBI Flush() and expect a successful response.
3.  Issue a gNOI Ping() test to ping itself and expect a successful response.
4.  Issue a P4RT `MasterArbitrationUpdate` with random `device_id`, and expect a `StreamMessageResponse` response (no need to check the correctness).
5.  Use Ondatra raw API to run CLI command on the DUT (e.g. show version). This validates that the SSH is working.
6.  Add `::/0` routing static routes configured in the management VRF, with default route preference and next hops being the adjacent IP of port-11 and port-12. 
```

Note: if "foregin-next-hop" is used in baseline setup, in step 1 we shall bring down eBGP sessions on port 11, port-12 from the ATE side.